tests/run-e2e.sh

```diff
@@ -1,3 +1,4 @@
+# TODO: add integration test for all deployment choices (minimum, estimator-only, server-only, full)
```

Comment:
@sunya-ch IMHO as a requirement, All major features must have an e2e before merging. We can of course create follow up PRs for testing edge cases but we must have e2e tests to exercise the green and the most common red path at least.

Reply:
@sunya-ch , now that #237  has merged, you should find the  `oc` package quite useful